sdk/lib/cxxrt/guard.cc

```diff
@@ -72,8 +72,9 @@ namespace
 		void unlock()
 		{
 			assert(high == LockBit);
-			high = 0;
-			futex_wake(&high, std::numeric_limits<uint32_t>::max());
+			high    = 0;
+			int res = futex_wake(&high, std::numeric_limits<uint32_t>::max());
+			assert(res >= 0);
```

Comment:
Please don't use `assert`, I don't think we use it anywhere else in the codebase.  It is there *purely* because C99 requires it, not because we should use it.

Reply:
Would you prefer a different spelling or merely discarding the result entirely?

Reply:
#404